dashboard/app/models/script.rb

```diff
@@ -605,12 +605,12 @@ def summarize
       peer_reviews_to_complete.times do |x|
         levels << {
             ids: [x],
-            position: 0,
             kind: 'peer_review',
             title: '',
             url: '',
             name: I18n.t('peer_review.reviews_unavailable'),
-            icon: 'fa-lock'
+            icon: 'fa-lock',
```

Comment:
I guess that's possible, but in that case I think we'd still want to show the default locked styling (so that the level being locked is more important that what type of level it is).